gem/templates/chhaajaa/core/tags/hero_article.html

```diff
@@ -7,31 +7,31 @@
       <ul class="teaser-stacklist">
         {% for article in articles %}
           <li class="teaser-stacklist__item">
-            {% if article.get_effective_image %}
-              <div class="teaser-stacklist__images">
-                <a href="{% pageurl article.specific %}" class="teaser-stacklist__anchor">
-                  {% image article.get_effective_image width-480 format-jpeg as teaser_thumbnail %}
-                  <img alt="{{ article.title }}" src="{{ teaser_thumbnail.url }}" class="teaser-stacklist__thumbnail"/>
-                </a>
-              </div>
-            {% else %}
-              {% for block in article.body %}
+            {% for block in article.body %}
+              {% if block.value.type == "video" %}
                 {% if block.value.youtube_link %}
                   <div class="teaser-stacklist__iframe">
                     {% embed block.value.youtube_link %}
                   </div>
+                  {% endif %}
+              {% else %}
+                {% if forloop.last and article.get_effective_image %}
```

Comment:
won't this display the video and the image? Like if it gets a block that is a video it will display it, and then if the next block isn't a video but is the last in the loop, it will display the image?